### DISCUSSION TITLE
`tea upgrade` update CLI to latest version

### DISCUSSION THREAD

**Discussion by mittalyashu** [pkgxdev/pkgx#679] [Ideas]
Add `tea upgrade` command to update to CLI to latest version instead of running this command `sh <(curl https://tea.xyz)` again and again.

**Comment by davdroman** [comment]
Agreed, this feels like a missing feature in my book.

**Comment by mxcl** [comment]
There is also `tea -S tea`.

I agree it being less explicit is weird.

**Reply by davdroman** [reply]
Whoa, I had no idea. Meta af. Thanks for the tip 👍
